### PR TITLE
(issue #107) Update desktop docs for cross-channel

### PR DESCRIPTION
**Pull request by flodolo**
Plus a few more nits and updates.

### CODE REVIEW

File: products/firefox_desktop/adding_nightly.md [hunk 1]
```diff
@@ -55,21 +44,43 @@ $ hg pull -u
...
+locales = [
+    "ach",
+    "af",
+    "an",
+    "ar",
+    "as",
+...
+```
+
+Identify the `locales` section, and add the new locale code between parenthesis, followed by a comma. As before, you can use Atom to make sure the list is in alphabetical order (make sure to select only the lines with actual locale codes before pressing `F5`).
```
Comment (Delphine) [inline products/firefox_desktop/adding_nightly.md]: Hmm "parenthesis" should be "quotes" (or "quotation marks", not sure anymore of the English term) here I believe :)

**Review by Delphine — CHANGES_REQUESTED** [review]
Just found one thing - rest looks good!

File: products/firefox_desktop/adding_nightly.md (hunk 1 above)
Comment (flodolo) [reply products/firefox_desktop/adding_nightly.md]: oh boy, those are definitely not parenthesis…

"double quotes"?

**Comment by flodolo** [comment]
Thanks, updated: https://github.com/mozilla-l10n/documentation/pull/113/commits/6c1dd5afb4ed71021ae93369fdae5933435bc65a

**Review by Delphine — APPROVED** [review]
I believe that works too :D LGTM now 🦄

**Pull request merged**